src/wmtk/MultiMeshManager.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include <tuple>
+#include "attribute/AttributeManager.hpp"
+#include "attribute/AttributeScopeHandle.hpp"
+#include "attribute/MeshAttributes.hpp"
+#include "Tuple.hpp"
+#include <spdlog/spdlog.h>
+
+namespace wmtk
+{
+class Mesh;
+class MultiMeshManager
+{
+    public:
+
+    MultiMeshManager();
+    ~MultiMeshManager();
+    MultiMeshManager(const MultiMeshManager& o);
+    MultiMeshManager(MultiMeshManager&& o);
+    MultiMeshManager& operator=(const MultiMeshManager& o);
+    MultiMeshManager& operator=(MultiMeshManager&& o);
+
+    //=========================================================
+    // Storage of MultiMesh
+    //=========================================================
+
+    bool is_parent_mesh() const { return m_is_parent_mesh; }
+    long child_id() const { return m_child_id; }
+    // helper function to read/write the map from source_tuple to target_tuple to the attribute
+    static void write_tuple_map_attribute(MeshAttributeHandle<long> map_handle, Mesh& source_mesh, const Tuple& source_tuple, const Tuple& target_tuple);
```

Comment:
this doesn't have to be static, and there should be an assert in this function that 
`&source_mesh.multi_mesh_manager == this`